### PR TITLE
[OPENY-12] Code block and Code block paragraph components decoupling

### PR DESCRIPTION
**Pull request by hamrant**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [x] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [x] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

## Steps for review

- [x] Login as admin
- [x] Go to /node/add/landing_page
- [x] Create landing_page with "OpenY paragraph code block" (use "Add code in paragraphs")
- [x] go to /admin/structure/block/block-content
- [x] check that you can see here created code block
- [x] go to /admin/modules/uninstall
- [x] uninstall "OpenY paragraph code block"
- [x] uninstall "OpenY Code Block" 
- [x] return to created landing_page, check that page works fine and not contain code block content (try to edit this page, all must be fine)
- [x] go to /admin/structure/block/block-content/types
- [x] check that "Code Block" not exist here
- [x] go to /admin/structure/block/block-content
- [x] check that you can't see here created code block
- [x] go to /admin/config/content/formats
- [x] check that "Code" text format not exist here
- [x] go to /admin/modules
- [x] enable "OpenY Code Block"  and "OpenY paragraph code block"
- [x] check that after modules enabling all functional, related to this modules works fine
- [x] check "Secondary Description and Sidebar" - after modules install it must contain "Code Block" in select list

### CODE REVIEW

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by hamrant** [comment]
test this please

**Comment by ddrozdik** [comment]
ok to test

**Comment by podarok** [comment]
ok to test

File: modules/openy_features/openy_block/modules/openy_code_block/openy_code_block.info.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: 'OpenY Code Block'
 description: 'OpenY Code Block'
+package: OpenY
 type: module
 core: 8.x
 dependencies:
```
Comment (podarok) [inline modules/openy_features/openy_block/modules/openy_code_block/openy_code_block.info.yml:6]: Bump module version, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_block_date/openy_prgf_block_date.info.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 name: 'OpenY Paragraph for Date Block'
 description: 'OpenY Paragraph shows content based on date and time (before, between, after).'
+package: OpenY
 type: module
 core: 8.x
 dependencies:
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_block_date/openy_prgf_block_date.info.yml:6]: Bump module version, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_code_block/openy_prgf_code_block.info.yml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 name: 'OpenY paragraph code block'
 description: 'Code block paragraph'
+package: OpenY
 type: module
 core: 8.x
 dependencies:
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_code_block/openy_prgf_code_block.info.yml:6]: Bump module version, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_location_by_amenities/openy_prgf_location_by_amenities.info.yml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 name: 'OpenY Paragraph Location By Amenities'
+package: OpenY
 type: module
 core: 8.x
 dependencies:
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_location_by_amenities/openy_prgf_location_by_amenities.info.yml:5]: Bump module version, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_lto/openy_prgf_lto.info.yml [hunk 5]
```diff
@@ -1,5 +1,6 @@
 name: 'OpenY Paragraph Limited Time Offer'
 description: 'Limited Time Offer Paragraph for Home Page based on Landing Page CT.'
+package: OpenY
 type: module
 core: 8.x
 dependencies:
```
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_lto/openy_prgf_lto.info.yml:6]: Bump module version, please

File: modules/openy_features/openy_taxonomy/modules/openy_txnm_amenities/openy_txnm_amenities.info.yml [hunk 6]
```diff
@@ -1,5 +1,6 @@
 name: 'OpenY Taxonomy Amenities'
 description: 'OpenY Taxonomy Amenities.'
+package: OpenY
```
Comment (podarok) [inline modules/openy_features/openy_taxonomy/modules/openy_txnm_amenities/openy_txnm_amenities.info.yml:3]: Bump module version, please

**Review by podarok — CHANGES_REQUESTED** [review]

File: modules/openy_features/openy_block/modules/openy_code_block/openy_code_block.info.yml (hunk 1 above)
Comment (ddrozdik) [reply modules/openy_features/openy_block/modules/openy_code_block/openy_code_block.info.yml:6]: @podarok do you mean to set there the same version of the module as the current OpenY version, or use something independent?

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build320 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build320 |
| Installation process | http://install.openy-dev.ffwua.com/build320/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/320/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build320/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build320/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build320/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build320/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build320/blogWCAG2AAhtmlcs.txt

**Comment by ddrozdik** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build324 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build324 |
| Installation process | http://install.openy-dev.ffwua.com/build324/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/324/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build324/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build324/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build324/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build324/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build324/blogWCAG2AAhtmlcs.txt

File: modules/openy_features/openy_block/modules/openy_code_block/openy_code_block.info.yml (hunk 1 above)
Comment (hamrant) [reply modules/openy_features/openy_block/modules/openy_code_block/openy_code_block.info.yml:6]: @podarok I have checked OpenY custom modules and features and they all not contain version... In this case I think better to create separated issue for all modules.

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build325 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build325 |
| Installation process | http://install.openy-dev.ffwua.com/build325/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/325/

**Comment by ddrozdik** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build348 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build348 |
| Installation process | http://install.openy-dev.ffwua.com/build348/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/348/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build348/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build348/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build348/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build348/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build348/blogWCAG2AAhtmlcs.txt

**Review by ddrozdik — APPROVED** [review]

**Pull request merged**